### PR TITLE
Full History Dataset Metadata

### PR DESCRIPTION
**Pull request by charlottedawson**
Fixes: #211   
_Creating metadata for the NZ Buildings Full History set_

### Change Description:
Metadata file created
...

### Notes for Testing:
Check consistency and wording makes sense. 
...

#### Source Code Documentation Tasks:
- [ ] README updated (where applicable)
- [ ] CHANGELOG (Unreleased section) updated
- [ ] Docstrings / comments included to help explain code

#### User Documentation Tasks:
- [ ] Confluence user guide updated (where applicable)

#### Testing Tasks:
- [ ] Double check for typos etc

### CODE REVIEW

File: metadata/nz_building_outlines_full_history.xml [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+        </gmd:RS_Identifier>
+      </gmd:referenceSystemIdentifier>
+    </gmd:MD_ReferenceSystem>
+  </gmd:referenceSystemInfo>
+  <gmd:identificationInfo>
+    <gmd:MD_DataIdentification>
+      <gmd:citation>
+        <gmd:CI_Citation>
+          <gmd:title>
+            <gco:CharacterString>NZ Building Outlines</gco:CharacterString>
```
Comment (dwsilk) [inline metadata/nz_building_outlines_full_history.xml]: Should be `NZ Building Outlines (Full History)`

File: metadata/nz_building_outlines_full_history.xml [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+• It does not contain any spikes;
+• It does not contain lines that are intersected with each other;
+• It can contain polygons with interior rings (holes);
+• It does not contain polygons with multiple exterior rings;
+• Building polygons captured will be greater than 10m².
+
+Please refer to the [Building Outlines Data Dictionary](https://nz-building-outlines.readthedocs.io/en/latest/published_data.html#table-nz-building-outlines-full-history) for detailed metadata and information about this layer.</gco:CharacterString>
+      </gmd:abstract>
+      <gmd:purpose>
+        <gco:CharacterString>This layer provides the building outlines for mainland New Zealand.</gco:CharacterString>
```
Comment (dwsilk) [inline metadata/nz_building_outlines_full_history.xml]: `provides the full history of building outlines for..`

File: metadata/nz_building_outlines_full_history.xml (hunk 1 above)
Comment (charlottedawson) [reply metadata/nz_building_outlines_full_history.xml]: done!

File: metadata/nz_building_outlines_full_history.xml (hunk 2 above)
Comment (charlottedawson) [reply metadata/nz_building_outlines_full_history.xml]: Done!

**Pull request merged**